packages/cli/src/util/generators.ts

```diff
@@ -4,24 +4,30 @@ import tree from "tree-node-cli";
 import { log } from "./serverLogger";
 
 export async function generateEmailsDirectory({
-  isTypescript,
   emailsDir,
+  isTypescript,
 }: {
-  isTypescript: boolean;
   emailsDir: string;
+  isTypescript: boolean;
 }) {
-  // copy the emails dir template in!
-  const path = `generator_templates/${isTypescript ? "ts" : "js"}/emails`;
   const srcDir =
     process.env.MM_DEV || process.env.NODE_ENV === "test"
       ? __dirname + "/.."
       : __dirname + "/../src";
-  await copy(resolve(srcDir, path), emailsDir, {
-    overwrite: false,
+
+  // copy the emails dir template in!
+  if (isTypescript) {
+    await copy(resolve(srcDir, "emails"), emailsDir, {
+      overwrite: false,
+    });
+  } else {
+    await copy(resolve(srcDir, "emails-js"), emailsDir, {
+      overwrite: false,
+    });
```

Comment:
you could DRY this and make it a little faster to read I think by using the a variable for the path and a single `copy` expression